django/db/models/fields/__init__.py

```diff
@@ -1500,6 +1500,8 @@ def __init__(self, verbose_name=None, name=None, max_digits=None,
                  decimal_places=None, **kwargs):
         self.max_digits, self.decimal_places = max_digits, decimal_places
         super(DecimalField, self).__init__(verbose_name, name, **kwargs)
+        self.validators.append(validators.DecimalValidator(self.max_digits,
+                                                           self.decimal_places))
```

Comment:
@timgraham The documentation states those two arguments as **required**, but they both get `None` as a default value, which means you can instantiate it without arguments: `DecimalField()`. Please correct me if I'm wrong.